src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/IslHelper.groovy

```diff
@@ -120,11 +126,16 @@ class IslHelper {
 
     def setAvailableBandwidth(Isl isl, long newValue) {
         cleanupManager.addAction(RESET_ISL_PARAMETERS, {resetAvailableBandwidth([isl, isl.reversed])})
+        database.updateIslMaxBandwidth(isl, newValue)
```

Comment:
added this call as almost every test has both calls of updating maxBandwidth+availableBandwidth (it seems logically to update both of these values).